server/routes/standardRouter.ts

```diff
@@ -0,0 +1,20 @@
+import { Router } from 'express'
+
+// const testMode = process.env.NODE_ENV === 'test'
+
+export default function standardRouter(): Router {
+  //
+  // // CSRF protection
+  // if (!testMode) {
+  //   router.use(csurf())
```

Comment:
will add back contents of the standard router once we have changed all the routes to use it